### DISCUSSION TITLE
Gallery of applications featuring ngx-extended-pdf-viewer

### DISCUSSION THREAD

**Discussion by stephanrauh** [stephanrauh/ngx-extended-pdf-viewer#780] [Show and tell]
Would you like to show your PDF project? I'm planning to add an exhibition of applications using this PDF viewer on https://pdfviewer.net, and this discussion is where it starts. Which project would you like to show?

**Comment by majora2007** [comment]
I'll add my application to the mix. Although it currently doesn't do much extra over your library, I do use it and plan to enhance it in a later release:

https://github.com/Kareadita/Kavita
https://www.kavitareader.com/

**Reply by stephanrauh** [reply]
Loving it already! Your showcase looks really nice. Keep up the good work! There's only one thing - and probably I was browsing too fast: I didn't find a PDF file on https://www.kavitareader.com/. Did I miss anything?

**Reply by majora2007** [reply]
That is because that's just the product website (it's free). You can see it in action on our demo:
https://demo.kavitareader.com/
Username: demouser
Password: Demouser64

**Reply by majora2007** [reply]
This is an exact url to a pdf book:
https://demo.kavitareader.com/library/1/series/17

**Comment by stephanrauh** [comment]
@AlexanderFalconi uses the PDF viewer in a website giving voters in Nevada more information about the judges they can vote:
https://www.ournevadajudges.com/judges/linda-marie-bell/cases?state=%7B%22show%22:%2250529-84697.pdf%22%7D
